src/mds/Server.cc

```diff
@@ -5489,6 +5489,8 @@ void Server::handle_client_setattr(const MDRequestRef& mdr)
       auto bl = req->get_data().cbegin();
       DECODE_START(1, bl);
       decode(header.change_attr, bl);
+      decode(header.file_offset, bl);
+      decode(header.block_size, bl);
```

Comment:
no verification that we actually got a block of `block_size`. Useful for debugging.

Reply:
Is this a comment that can be resolved or do you request an action from this?